src/Commands/Command.php

```diff
@@ -50,7 +50,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
             if ($invocation['status'] === 'success') {
                 IO::spinClear();
-                IO::writeln($invocation['output']);
+                $output->writeln($invocation['output']);
```

Comment:
Any reason for this change? We need to use `IO::writeln()` to account for all other interactive outputs.